src/lib/init.js

```diff
@@ -49,10 +49,25 @@ export function init(configUpdates) {
 
 				// set cookies on digitrust domain after consent submitted
 				const { addEventListener, getVendorConsents } = cmp.commands;
+				if (config.digitrust.pixel) {
+					addEventListener('consentStringUpdated', digitrustPixel);
+				}
 				if (config.digitrust.redirects) {
 					addEventListener('consentStringUpdated', digitrustRedirect);
 				}
 
+				function digitrustPixel() {
+					getVendorConsents([64], (result) => {
```

Comment:
The same getVendorConsent call is used in the digitrustRedirect function. Why wouldn't we create a function which  calls the getVendorConsents and checks this condition:

`result && result.vendorConsents && result.vendorConsents[64]`

Reply:
created `getDigiTrustVendorConsents`